assets/static/js/entity/ResearchGroup.js

```diff
@@ -39,6 +39,9 @@ $(document).ready(function()
     //Disable RIS ID if not in create mode
     if ($("form[entityType=\"ResearchGroup\"] #id").val() !== "") {
         $("form[entityType=\"ResearchGroup\"] #id").attr("readonly",true);
+    } else {
+        var nextId = $("#id").attr("next-id");
+        $("input[name='id']").val(nextId);
```

Comment:
Use the same selector as above, your can just copy the selector above, and set it to a variable.